### PR TITLE
(IO-2) Extending Command Manager window + Code cleanup

### PR DESCRIPTION
**Pull request by Lecrut**

### CODE REVIEW

File: src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java [hunk 1]
```diff
@@ -13,7 +13,6 @@
 import edu.kis.powp.jobs2d.Job2dDriver;
 import edu.kis.powp.jobs2d.command.DriverCommand;
 import edu.kis.powp.jobs2d.command.manager.CommandManager;
-import edu.kis.powp.jobs2d.drivers.DriverManager;
 import edu.kis.powp.jobs2d.drivers.adapter.LineDriverAdapter;
 import edu.kis.powp.jobs2d.features.CommandsFeature;
 import edu.kis.powp.jobs2d.features.DriverFeature;
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java:18]: DriverFeature depends on DriverManager so there are no changes in the coupling.

**Review by kowallus — CHANGES_REQUESTED** [review]
Is this fix a joke?

File: src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java (hunk 1 above)
Comment (Jvsin) [reply src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java:18]: okey, sory I missunderstood.

File: src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java [hunk 2]
```diff
@@ -91,6 +114,10 @@ private void clearCommand() {
         updateCurrentCommandField();
     }
 
+    private void runCommand(){
+        commandManager.getCurrentCommand().execute(driverManager.getCurrentDriver());
+    }
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java:119]: go for independency from DriverManager. Move this method to CommandManager and extract CommandManager interface (DIP principle)

**Review by kowallus — CHANGES_REQUESTED** [review]
try again with DIP

File: src/main/java/edu/kis/powp/jobs2d/command/manager/CommandManager.java
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/manager/CommandManager.java:1]: Extract interface from CommandManager (DIP principle)! You can name it e.g.,
public interface ICommandManager { ... 

Then provide CommandManager instances to other objects using Dependency Injection.

**Review by kowallus — CHANGES_REQUESTED** [review]
You have still problem with understanding how to extract an interface and apply DIP & DI.

**Review by kowallus — APPROVED** [review]
CommandFeature still holds the indirect dependency to CommandManager, but lets move on!

**Pull request merged**